app/src/main/java/com/devshish/internship/presentation/ui/player/PlayerFragment.kt

```diff
@@ -81,12 +85,35 @@ class PlayerFragment : Fragment(R.layout.fragment_player) {
             tvSong.text = song.title
             tvArtist.text = song.artist
             tvDurationEnd.text = convertMillisToTime(song.duration)
-            sPlayer.max = song.duration
+            slider.valueFrom = 0f
+            slider.valueTo = song.duration.toFloat()
 
             Glide.with(this@PlayerFragment)
                 .load(song.imageUri)
                 .placeholder(R.color.black)
                 .into(ivSongCover)
         }
     }
+
+    override fun onCreateOptionsMenu(menu: Menu, inflater: MenuInflater) {
```

Comment:
It will be better if you inflate menu anyway, and just change the visibility of items 

Like this: (but you need to make this item invisible in xml by default)
![image](https://user-images.githubusercontent.com/23121846/143847641-4e87f10a-3d93-47ad-af4f-8be17a2d00c0.png)

Reply:
Still not correct